azkaban-common/src/main/java/azkaban/executor/AbstractExecutorManagerAdapter.java

```diff
@@ -75,7 +91,7 @@ public long getAgedQueuedFlowSize() {
       size = this.executorLoader.fetchAgedQueuedFlows(Duration.ofMinutes(minimum_age_minutes))
           .size();
     } catch (final ExecutorManagerException e) {
-      logger.error("Failed to get flows queued for a long time.", e);
+      this.logger.error("Failed to get flows queued for a long time.", e);
```

Comment:
logger seems to be declared private static final. Why this is needed?

Reply:
Fixed it.